packages/env/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 -   `run` command now has a `--env-cwd` option to set the working directory in the container for the command to execute from.
 
+### Bug fix
+
+-   Docker containers now run as the host user. This should resolve problems with permissions arising from different owners
```

Comment:
Thoughts on marking this as a breaking change instead?

Reply:
I think that's fair!